src/NServiceBus.Persistence.AzureStorage.AcceptanceTests/Sagas/When_doing_request_response_between_sagas.cs

```diff
@@ -54,7 +54,7 @@ protected override void ConfigureHowToFindSaga(SagaPropertyMapper<RequestRespons
 
                 public class RequestResponseRequestingSagaData : ContainSagaData
                 {
-                    public virtual Guid CorrIdForResponse { get; set; } //wont be needed in the future
+                    public virtual Guid CorrIdForResponse { get; set; }
```

Comment:
Looking at the `ConfigureHowToFindSaga` code, `CorrIdForResponse` property on the saga data is used to map the incoming messages. Which means by default, `Data.CorrIdForResponse` is assigned by NSB. Why the comment in place says "won't be needed in the future" now puzzles me as well. This is used to perform the correlation and w/o the endpoint simply won't work (ie test will fail).

Saying that, there's really no test. I can't see any `[Test]` or `Should_`.  What is my blind point?! :confused: